docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
+. On the left side of the screen, click the *{ProductName}* tab.
++
+If this is your first configuration, the run configuration panel is displayed on the right.
+
+. If this is not your first configuration, right-click on one of the configurations in the list and select *New configuration*.
++
+The run configuration panel is displayed on the right.
```

Comment:
Is "run configuration" a GUI element? (Should it be bold?)

Reply:
The element is unnamed, so I didn't bold it. I'm following the guidance Avital gave for the VS Code guide -- both UIs have named and unnamed elements. I've bolded he named elements and left the unnnamed elements in plain text. I'd prefer to keep the style consistent over the guides. if you feel strongly it should be changed, I will have to change other guides in a separate PR. I think the convention i'm using is a good choice, but it's your call.

Reply:
OK, just asking. Sounds good. I'm just wondering how the user will understand what you're talking about considering that it's unnamed. I'm guessing that it's obvious enough.